company/team/index.md

```diff
@@ -2,18 +2,16 @@
 
 This page contains brief bios of our team. Teammates may also have a personal documentation page in this directory that is named according to their Sourcegraph email address (e.g. you@sourcegraph.com -> you.md).
 
-<!--
-
 To add yourself to this page, copy the following template, paste it at the end of this file, and make it about yourself!
 
 ## Full Name (optional pronouns)
 
+- Name pronunciation: first name phonetic spelling
```

Comment:
Some people might interpret this literally and use the [phoenetic alphabet](https://en.wikipedia.org/wiki/International_Phonetic_Alphabet) (which I don't think would help anyone actually pronounce someone's name.

```suggestion
- Describe how to pronounce your name
```